scripts/copyFiles.mjs

```diff
@@ -157,12 +157,7 @@ async function run() {
   try {
     const packageData = await createPackageFile();
 
-    const filesToCopy = [
-      './README.md',
-
-      // TODO: Remove `grid` folder to flatten the structure
-      packageData.name.includes('grid') ? '../../../CHANGELOG.md' : '../../CHANGELOG.md',
-    ];
+    const filesToCopy = ['./README.md', '../../CHANGELOG.md'];
```

Comment:
When I migrated to use the same bundling strategy as the core, this code you just removed was the only diff with the core file
So maybe you could just remove everything from this file and import the core one, like we are doing for `buildTypes.mjs`

But this need to be double checked of course

Reply:
I'll look into this and submit a follow-up PR if necessary

Reply:
Unfortunately, there are other non-trivial differences between our script and the one from the core repo.
Making the `copyFiles` script generic would be a significant effort and I think it's not worth it.
But I think it makes sense to at least reuse some functions from that script.
I'll submit a PR to the core repo.

Reply:
https://github.com/mui/material-ui/pull/40970